scripts/bagging_variables.sh

```diff
@@ -12,12 +12,14 @@ export BAG_ALWAYS="/odom /absodom /clock /tf /tf_static"
 # Define the directory that the bags will be stored in
 export BAG_DIR=~/bags
 
+# Which camera image type to use
+export CAM_SUFFIX="image_rect_color"
```

Comment:
If you want to add that flag (or, better, arbitrary variables like CAM_SUFFIX) it would be cool. Until than it was just a way to save me typing the same thing twice and we shouldn't overthink it. I'll change the variable name and comment.